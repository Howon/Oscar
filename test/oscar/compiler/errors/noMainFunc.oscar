message testNow()


actor Tester() {

  receive = {
    | testNow() => {
        Println("Error1");
      }
    | die() => {
        }
  }
}
