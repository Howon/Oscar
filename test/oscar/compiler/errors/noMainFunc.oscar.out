Error: No main function found in this program
